.gitignore

```diff
@@ -6,6 +6,7 @@
 .DS_Store
 .idea
 *.code-workspace
+go.work.sum
```

Comment:
I think the .sum file should be checked in. There is a reason to ignore `go.work` in some cases (not our case) but if `go.work` is checked in, `go.work.sum` should also be checked in. Its the same as `go.mod` and `go.sum`

Reply:
I think in this case it actually makes sense to not include it as this is just for the local setup. If I understand correctly the data is collected from the modules mod files anyway. Therefore IMHO this would be just an unnecessary source for merge conflicts.